src/attack.c

```diff
@@ -1074,15 +1074,32 @@ attack_network (struct scan_globals *globals, kb_t *network_kb)
     }
 
   /* Initialize the attack. */
+  int plugins_init_error = 0;
   sched = plugins_scheduler_init (prefs_get ("plugin_set"),
                                   prefs_get_bool ("auto_enable_dependencies"),
-                                  network_phase);
+                                  network_phase, &plugins_init_error);
   if (!sched)
     {
       g_message ("Couldn't initialize the plugin scheduler");
       return;
     }
 
+  if (plugins_init_error > 0)
+    {
+      char buf[96];
+      int i = atoi (prefs_get ("ov_maindbid"));
+      kb_t main_kb = NULL;
+
+      sprintf (buf,
+               "%d errors were found during the plugin scheduling. "
+               "Some plugins will not be launch.",
```

Comment:
It suppose it should be "Some plugins will not be launched." or "Some plugins have not been launched."